gpiozero/pins/pigpiod.py

```diff
@@ -68,10 +71,65 @@ class PiGPIOPin(Pin):
 
     .. _pigpio: http://abyz.co.uk/rpi/pigpio/
     """
+    def __init__(
+            self, host=os.getenv('PIGPIO_ADDR', 'localhost'),
+            port=int(os.getenv('PIGPIO_PORT', 8888))):
+        super(PiGPIOFactory, self).__init__()
+        self.pin_class = PiGPIOPin
+        self.spi_hardware_class = PiGPIOHardwareSPI
+        self.spi_software_class = PiGPIOSoftwareSPI
+        self.shared_spi_hardware_class = PiGPIOHardwareSPIShared
+        self.shared_spi_software_class = PiGPIOSoftwareSPIShared
+        self._connection = pigpio.pi(host, port)
+        self._host = host
+        self._port = port
+        self._spis = []
```

Comment:
Not really much point given that (once I've fixed up the prior comment about the missing conflicts ... oops!) duplicate SPI interfaces won't be able to be constructed anyway.